src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -3580,12 +3598,12 @@ namespace AirflowNetworkBalanceManager {
 
                 // Surface exposure fraction
                 if (Numbers(2) > 1) {
-                    ShowContinueError("Duct surface exposure fraction greater than 1. Check input in: " + CurrentModuleObject + " " +
+                    ShowWarningError("Duct surface exposure fraction greater than 1. Check input in: " + CurrentModuleObject + " " +
```

Comment:
Here and below, several existing warnings incorrectly started with `ShowContinueError`.